### ISSUE TITLE
ToView deprecated 

### ISSUE THREAD

**Issue by bigteejay** [dotnet/EntityFramework.Docs#4239]
If I switch the documentation to 3.1 (or later) I will see the following notification for [ToView](https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.relationalquerytypebuilderextensions.toview?view=efcore-2.2)...

> "The requested page is not available for Entity Framework Core 3.1. You have been redirected to the newest product version this page is available for."

It may be mentioned in an upgrade guide, but it would be helpful if this page pointed out this method is deprecated in some fashion no matter the version (so devs can have a heads up that it creates a maintenance pain point).

PS:  Wish I could edit title as it was meant as a question (and so as to not mislead/confuse the matter).  Technically, it does not appear to be "deprecated".

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 30f78bb4-03b8-e3af-d656-3a6c8a1c29cf
* Version Independent ID: a27dcfdb-8dae-6838-611d-0669b19ab841
* Content: [RelationalQueryTypeBuilderExtensions.ToView Method (Microsoft.EntityFrameworkCore)](https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.relationalquerytypebuilderextensions.toview?view=efcore-2.2&viewFallbackFrom=efcore-3.1)
* Content Source: [dotnet/xml/Microsoft.EntityFrameworkCore/RelationalQueryTypeBuilderExtensions.xml](https://github.com/dotnet/EntityFramework.ApiDocs/blob/live/dotnet/xml/Microsoft.EntityFrameworkCore/RelationalQueryTypeBuilderExtensions.xml)
* Product: **ef-core-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **avickers**

**Comment by bigteejay** [comment]
UPDATE:  I just found [this related issue](https://github.com/dotnet/efcore/issues/14787) filed under dotnet.

TLDR:  ToView initially removed and left as potentially being designed back in.

**Comment by ajcvickers** [comment]
I don't think there is anything we can do here. The docs platform does not support doing something different, and the breaking change was documented: https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.x/breaking-changes#qt
